docs/topics/db/transactions.txt

```diff
@@ -325,6 +325,15 @@ If that hypothetical database write is instead rolled back (typically when an
 unhandled exception is raised in an :func:`atomic` block), your function will
 be discarded and never called.
 
+It's sometimes useful to register callback functions that can fail. Passing
+``is_robust=True`` allows the next functions to be executed even if the current
+function throws an exception. All errors derived from Python's ``Exception``
+class are caught and logged to the ``django.db.backends.base`` logger.
```

Comment:
`docs/ref/logging.txt` contains docs for all logging that Django does, afaik, so it would be good to add a section there too.

Reply:
> `docs/ref/logging.txt` contains docs for all logging that Django does, afaik, ...

We have several undocumented loggers i.e. `django.utils.autoreload`, `django.dispatch`, `django.db.models`, and `django.contrib.gis`.

> ... so it would be good to add a section there too.

Only folks that use `robust=True` will use this logger, so, IMO it's enough to mention it here.